x/escrow/init_test.go

```diff
@@ -0,0 +1,80 @@
+package escrow
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"testing"
+
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/x/cash"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/store"
+)
+
+func TestGenesisKey(t *testing.T) {
+	const genesis = `
+{
+  "escrow": [
+    {
+      "amount": [
+        {
+          "ticker": "ALX",
+          "whole": 987654321
+        },
+        {
+          "ticker": "IOV",
+          "whole": 123456789
+        }
+      ],
+      "arbiter": "cond:foo/bar/636f6e646974696f6e64617461",
```

Comment:
Cool, this is definitely more readable, makes clear which extension it is relying on